src/main/asciidoc/intro.adoc

```diff
@@ -0,0 +1,3 @@
+= Intro
+
+Ehcache 3.0.0 has been released!  In this post I would like to demonstrate how to use it as Spring's caching implementation.  This article will show you how to use Ehcache 3.0.0, Spring 4.2 and the JSR-107 annotations to build a simple Spring web application.  I have also included some background reading for those of you that would like to read more in depth on the subject matter.
```

Comment:
3.0.1 will be out in a couple of days.  Suggest you just say "Ehcache 3 has been released!"